Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient.m

```diff
@@ -424,75 +420,35 @@ - (NSURLSessionDataTask *)getCheckout:(BUYCheckout *)checkout completion:(BUYDat
 
 - (NSURLSessionDataTask *)updateCheckout:(BUYCheckout *)checkout completion:(BUYDataCheckoutBlock)block
 {
+	NSAssert(checkout, @"Failed to update checkout. Checkout must not be nil");
+	NSAssert([checkout hasToken], @"Failed to update checkout. Checkout must have a valid token associated with it/");
```

Comment:
Typo at the end of the assert comment. I think we can get away with just the second assert, as the first is implicit in it.